.github/workflows/test-runner-nightly-integration.yml

```diff
@@ -1,25 +1,57 @@
 # This workflow will build a Java project with Gradle
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
 
-name: Test Runner Integration Tests
+name: Test Runner Nightly Integration Tests
 
 on:
-  push:
-    branches: [ dev ]
-  pull_request:
-    branches: [ '**' ]
+  workflow_dispatch: {}
```

Comment:
If I understand right, I think we want to run the tests locally on push to PR (the same trigger we had before). The nightly "push to test environment -> run tests" workflow is also important, but the test environment isn't set up quite yet.

Reply:
Fixed trigger to `push`.